mesheryctl/internal/cli/root/system/channel.go

```diff
@@ -223,6 +223,12 @@ var channelCmd = &cobra.Command{
 	Example: `
 // Subscribe to release channel or version
 mesheryctl system channel [stable|stable-version|edge|edge-version]
```

Comment:
We can probably update this documentation too. I don't think `mesheryctl system channel` itself accepts any args.

Reply:
```suggestion
// Subscribe to release channel or version
mesheryctl system channel 
```

Reply:
// @legendarykamal ⬆️

Reply:
ok yes @nishantwrp I agree it doesn't accept on itself , ok @alphaX86 I will update it now.